pages/index.js

```diff
@@ -1,21 +1,33 @@
 import Head from "next/head";
 import styled from "styled-components";
 import { Inter } from "next/font/google";
+import { useRouter } from "next/router";
+import useSWR from "swr";
 
 const inter = Inter({ subsets: ["latin"] });
 
-export default function Home() {
+export default function Home({ name, location, image, bewerten, preis }) {
```

Comment:
```suggestion
export default function Home() {
```
Warum destructurest du hier diese keys? Du verwendest sie nirgenswo und sie passen auch gar nihct zu deiner Daten strukture, da currency und id fehlt. Kannst du weglassen